index.html

```diff
@@ -185,5 +185,25 @@ <h2 class="third-text"><a>let's connect</a></h2>
           </li>
         </ul>
     </section>
+    <section class="contact-form">
+        <ul>
+            <li><span class="ellipse-12-2"></span></li>
+            <li><span class="ellipse-5-w"></span></li>
+            <li><span class="ellipse-3-w"></span></li>
+            <li><span class="rect-4-d"></span></li>
+            <li><span class="rect-7-2"></span></li>
+        </ul>
```

Comment:
- [ ] This could be repetitive, but all tools are important where we build a website, the last review @MahmoudBakr23  talk about use `.section { background-image : URL("image_path_here=)`. Doing this will help you to load only one image in place to load 5 diff images, for memory cost is an improvement and I'm sure that would make your app better for speed. I kindly invite you to add all these images in only one and then,  add it again. 🐶

- This is the last point to complete this, just do it and you'll be on the other side.